akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpMessageParser.scala

```diff
@@ -243,6 +243,7 @@ private[http] trait HttpMessageParser[Output >: MessageOutput <: ParserOutput] {
           case ';' if cursor > offset ⇒ parseChunkExtensions(size.toInt, cursor + 1)()
           case '\r' if cursor > offset && byteChar(input, cursor + 1) == '\n' ⇒ parseChunkBody(size.toInt, "", cursor + 2)
           case '\n' if cursor > offset ⇒ parseChunkBody(size.toInt, "", cursor + 1)
+          case c if CharacterClasses.WSP(c) ⇒ parseSize(cursor + 1, size)
```

Comment:
perhaps we should add some comment that we know this is illegal in the spec but we've seen it in the wild?